crates/rolldown/src/utils/render_chunks.rs

```diff
@@ -18,12 +18,12 @@ pub async fn render_chunks<'a>(
     if let InstantiationKind::Ecma(ecma_meta) = &asset.kind {
       let render_chunk_ret = plugin_driver
         .render_chunk(HookRenderChunkArgs {
-          code: asset.content.clone(),
+          code: asset.content.clone().try_into_string()?,
```

Comment:
Why do we need to call the hook if it content is none string? Did rollup do that?

Reply:
I used a more proper method in https://github.com/rolldown/rolldown/pull/2560.